tests/functional/TestKubeSmokeTest/testHeketiRpc.sh

```diff
@@ -90,7 +114,6 @@ setup_all_pods() {
 
 	echo -e "\nStart gluster mock container"
 	start_mock_gluster_container 1
-	start_mock_gluster_container 2
 }
 
 test_peer_probe() {
```

Comment:
The test is not to test daemonsets, the test is for Heketi to find the pod who has been deployed by the model as a daemonset.